src/events/message/commands/help/link.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from "zod";
+import { CommandArg, Message, word } from "../types";
+
+// "?link <topic>" provides a simple shortcut to various useful URLs
+
+const LINKS: { [k: string]: string } = {
+  docs: "https://docs.codewars.com/",
+  honor: "https://docs.codewars.com/gamification/honor",
+  katas: "https://www.codewars.com/kata/latest/my-languages",
+  authoring: "https://docs.codewars.com/authoring",
+  ranking: "https://docs.codewars.com/curation/kata",
+  troubleshooting: "https://docs.codewars.com/training/troubleshooting",
+  github: "https://github.com/codewars/",
```

Comment:
If you want to link https://github.com/codewars/codewars.com/issues maybe add `issues`? 

Again, we can tweak this later too. I can merge this if you're ready.

Reply:
I think for now I'm ready. We can add more as we notice the need arise.